### ISSUE TITLE
Minor issues with `fread`

### ISSUE THREAD

**Issue by ksens** [Paradigm4/ArrayOpR#12]
- [ ] On a new R session, I tried the following
```R
library(arrayop) # optional step to leave out the `arrayop::` prefix.
creds_file = '~/.scidb_auth_secure_user2'
conn = db_connect(
  host = "localhost", 
  username = jsonlite::read_json(creds_file)[["user-name"]], 
  token = jsonlite::read_json(creds_file)[["user-password"]])

xx = conn$fread(
  "/flexfs/srikant/data/feature_metadata_expresso/uniprot/current_release/knowledgebase/idmapping/idmapping.dat.example", 
  header = F, nrow = 10) # infer column type using only up to first 10 rows
```
... and got an error:

```
"%>%" not defined
```

This was resolved by invoking:

```R
library(magrittr)
```

- [x] In the above code block, the `nrow = 10` parameter seems to have no effect

**Comment by smoore-p4** [comment]
The nrow = 10 is saying how many rows to look into to file to infer types, not how many rows to ingest into scidb. (so it isn't super obvious when it's being used, other than time it takes to figure out what the schema should look like)
Let me look into the %>% because that hasn't been an issue for me, but I might have something else loaded first.

**Comment by ksens** [comment]
Ah, got it. I should have read the docs more carefully 😄 

Thanks for the clarification.

**Comment by smoore-p4** [comment]
I'm putting in a fix to remove the requirement for `library(magrittr)` for fread. I'm thinking we'll do a new tagged version soon with a bunch of non-breaking improvements soon.

**Comment by smoore-p4** [comment]
Fixed with #19
